guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: guide
+title: Transaction privacy
+description: An overview of how to help users maintain their financial privacy while using Bitcoin.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/privacy/privacy-preview.jpg
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/privacy/header.jpg"
+    retina = "/assets/images/guide/payments/privacy/header@2x.jpg"
```

Comment:
That image is 5MB. Generally, the images are huge. In a quick test via [ImageOptim](https://imageoptim.com/mac) I was able to reduce the size by almost 90%.

Reply:
hmm ive stopped using image optim and just relied on the ruby plugin these days. it ran, so it seems like its still not the most optimal solution.

Reply:
One thing to keep in mind is that images are only optimized the first time they are added. If you add an image, it gets compressed, and then you overwrite that image with a new export, it does not get compressed again. Could that be it?

Reply:
aha. ye i think that may be it actually.